Fabric.Identity.API/scripts/CatalystDosIdentity.psm1

```diff
@@ -0,0 +1,620 @@
+<#
+    .Synopsis
+    Attempts to get an access token for given client
+
+    .Description
+    Takes in a client id and secret with valid scope and hits the identity server provided and returns an access token.
+
+    .Parameter identityUrl
+    The identity url to get the access token from
+
+    .Parameter clientId
+    The id of the client the access token is being requested for
+
+    .Parameter secret
+    The secret of the client the access token is being requested for
+
+    .Parameter scopes
+    The scope of the access token to be requested
+
+    .Example
+    Get-AccessToken -identityUrl "https://server/identity" -clientId "sample-client-id" -secret "SECrEtStrING" -scopes "offline_access"
+#>
+function Get-AccessToken {
+    param(
+        [Parameter(Mandatory = $True)] [Uri] $identityUrl,
+        [Parameter(Mandatory = $True)] [string] $clientId,
+        [Parameter(Mandatory = $True)] [string] $secret,
```

Comment:
I didn't catch this in the initial review - but perhaps this parameter should be securestring?